graylog2-web-interface/src/preflight/components/ConfigurationWizard/CAUpload.tsx

```diff
@@ -148,7 +168,7 @@ const CAUpload = () => {
                        type="password"
                        label="Password" />
           <UnsecureConnectionAlert renderIfSecure={<Space h="md" />} />
-          <Button disabled={isSubmitting || !isValid} type="submit">
```

Comment:
Since `isSubmitting` will be set to `false` after the promise got resolved, with https://github.com/Graylog2/graylog2-server/pull/16844/files#diff-7e84080ac8f664e0561a74f947aaa5c4a58d9454b5cd0c492e943a04b7da9066L92, do we still want to remove `isSubmitting` here?

Reply:
Yes, it helps testing this PR.